src/App.js

```diff
@@ -131,27 +117,35 @@ export class DatasetEditor extends React.Component {
 
   mapDatasetToFiscalFormat = (resource) => {
     const dataset = { ...this.state.dataset };
-
-    resource.schema.fields.forEach((f) => {
-      f.type = f.columnType;
-      delete f.columnType; //os-types requires type to be of rich type and will not accept the property colunmType
-    });
-    let fdp = new TypeProcessor().fieldsToModel(resource["schema"]["fields"]);
-
-    if (!Object.keys(dataset).includes("model")) {
-      dataset.model = fdp.model;
+    let model;
+    let schema;
+    if (dataset.resources.length > 1) {
```

Comment:
It's a subtle one, but it's correct. An uploaded resource gets added to the data package before this test takes place. So at any given time, before we get here, a resource already exist but it may not have a schema and model

Reply:
Oh great, thanks for taking the time to shoot me an explanation, that makes sense. :+1: